### PR TITLE
Rephrase warning message for strictly confined snap

### PR DESCRIPTION
**Pull request by anvial**
This PR rephrases a warning message if you install juju in a strictly confined Snap.

## Checklist

- [x] Code style: imports ordered, good names, simple structure, etc
- [x] Comments saying why design decisions were made
- [x] Go unit tests, with comments saying what you're testing

## QA steps

```sh
// Nothing
```

## Links

**Jira card:** [JUJU-[4693]](https://warthogs.atlassian.net/browse/JUJU-4693)

### CODE REVIEW

File: cmd/juju/commands/main.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func (m jujuMain) Run(args []string) int {
 	if err = juju.InitJujuXDGDataHome(); err != nil {
 		cmd.WriteError(ctx.Stderr, err)
 		if errors.Is(err, errors.NotFound) {
-			ctx.Warningf("Installing juju from a snap, the .local/share directory should be manually created as it is strictly confined.")
+			ctx.Warningf("Installing Juju in a strictly confined Snap. To ensure correct operation, create the .local/share directory manually.")
```
Comment (jnsgruk) [inline cmd/juju/commands/main.go:150]:
```suggestion
			ctx.Warningf("Installing Juju in a strictly confined Snap. To ensure correct operation, create the ~/.local/share/juju directory manually.")
```

**Review by manadart — APPROVED** [review]
Thanks for this. Please make Jon's suggested change before landing.

File: cmd/juju/commands/main.go (hunk 1 above)
Comment (anvial) [reply cmd/juju/commands/main.go]: thx

**Pull request merged**